restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/AttributeValidator.java

```diff
@@ -0,0 +1,548 @@
+package org.restcomm.connect.interpreter.rcml;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.log4j.Logger;
+
+public interface AttributeValidator {
+
+	public boolean RecordValidator (Tag tag);
+	public boolean DialValidator (Tag tag);
+	public boolean EmailValidator (Tag tag);
+	public boolean PauseValidator (Tag tag);
+	public boolean SmsAndFaxValidator (Tag tag);
+	public boolean GatherValidator (Tag tag);
+	public boolean PlayAndSayValidator (Tag tag);
+	
+	
+}
+
+ class Validator implements AttributeValidator{

```

Comment:
i dont like having interface and impl class in same file,but this is just cosmetic